app/views/onboardings/_task_card.html.erb

```diff
@@ -0,0 +1,43 @@
+<script>
+  function closeTaskCard() {
+    var taskCard = document.getElementsByClassName("onboarding-task-card")[0];
+    taskCard.style.display = "none";
+
+    localStorage.setItem('task-card-closed', 'yes');
+  }
+</script>
+
+<div class="onboarding-task-card">
+  <button class="close" onclick="closeTaskCard()">
+    <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M6.99974 5.58623L11.9497 0.63623L13.3637 2.05023L8.41374 7.00023L13.3637 11.9502L11.9497 13.3642L6.99974 8.41423L2.04974 13.3642L0.635742 11.9502L5.58574 7.00023L0.635742 2.05023L2.04974 0.63623L6.99974 5.58623Z" fill="white" />
+    </svg>
+  </button>
+
+  <div>
+    <%= inline_svg_tag("staggered-dev.svg", aria: true, title: "staggered dev stickers") %>
+  </div>
+
+  <h1 class="task-card-title">You're now apart of the community</h1>
```

Comment:
```suggestion
  <h1 class="task-card-title">You're now a part of the community!</h1>
```
the `!` is a suggestion; i'm not sure what the actual copy says but we should include some ending punctuation here

Reply:
I believe the copy includes an exclamation point if I'm not mistaken, so good catch @vaidehijoshi . 😄